### ISSUE TITLE
Unable to bring up the application per instructions

### ISSUE THREAD

**Issue by Asuza** [TailorDev/crick#115]
Running `make dev` in the _/api_ folder seems to be successful.
Running `make dev` in the _/web_ folder results in the following:
```
~/dev/crick/web · (master)
⟩ make dev
yarn start
yarn run v1.15.2
$ react-scripts start
Could not find an open port at crick.dev.
Network error message: listen EADDRNOTAVAIL 68.183.147.13

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
make: *** [Makefile:4: dev] Error 1
```
Is there a step I'm missing somewhere? Thanks.

**Comment by jmaupetit** [comment]
Hi @Asuza!

You are right: it's mentioned in the docs!

You need to configure your system to resolve `.dev` TLD as `localhost` (`127.0.0.1`). We use `dnsmasq` to do so. You will find relevant tutorial over the web depending on your OS [1, 2].

Hope this helps.

[1] https://passingcuriosity.com/2013/dnsmasq-dev-osx/
[2] https://ziyan.jdeen.com/systems/2018/06/09/local-development-dot-test-localhost.html

**Comment by willdurand** [comment]
I think another yet related problem is that `.dev` is now a reserved TLD :/ (owned by Google)

**Comment by jmaupetit** [comment]
Yes, and I stumbled upon it while trying to visit https://github.dev ... 😓

**Comment by Asuza** [comment]
@jmaupetit where is it mentioned in the docs? I looked all over (again, after reading your comment) and still couldn't find mention of it. Thank you!

**Comment by jmaupetit** [comment]
@Asuza It's not :cry:
